src/Nodes/NewNode.tsx

```diff
@@ -34,35 +37,64 @@ const FormItem = styled.div`
     &:last-child {
         border-bottom: ${itemBorder};
     }
+    &.error label {
+        color: red;
+    }
 `
 
 const NewNode = () => {
     const [loading, setLoading] = useState(false)
+    const nodeRef = gun.get(
+        namespace + 'node'
+    ) /*is node (noun) plural? ;) #trickledown42*/
```

Comment:
I'm getting this error. 
![image](https://user-images.githubusercontent.com/3623016/146656836-bef3f8fd-caa5-4660-80bb-60ad6951b418.png)

Reply:
not seeing this with the latest commit. more investigation needed for hashing in Gun!